docs/kpod-login.1.md

```diff
@@ -0,0 +1,67 @@
+% kpod(1) kpod-login - Simple tool to login to a registry server
+% Urvashi Mohnani
+# kpod-login "1" "August 2017" "kpod"
+
+## NAME
+kpod-login - Login to a container registry
+
+## SYNOPSIS
+**kpod login**
+**SERVER**
+[**--help**|**-h**]
```

Comment:
--outfile, -o

Reply:
--user
--password

Reply:
I thought these came later on and not right under synopsis.

Reply:
We need to be consistent, they either come in both places or only later.

Reply:
okay, fixed.